pkg/cluster/mpi/statefulset.go

```diff
@@ -34,6 +34,26 @@ var (
 	workerCommand = []string{
 		launchScriptPath,
 	}
+	sidecarPorts = []corev1.ContainerPort{
+		{
+			Name:          rsyncPortName,
+			ContainerPort: rsyncPort,
+		},
+	}
+	sidecarProbe = &corev1.Probe{
+		Handler: corev1.Handler{
+			TCPSocket: &corev1.TCPSocketAction{
+				Port: intstr.FromInt(rsyncPort),
+			},
+		},
+	}
+	sidecarCommand = []string{
+		"/bin/bash",
+		"-c",
+		"/usr/bin/ssh-keygen -f /opt/domino/etc/ssh/ssh_host_key -N '' -t ecdsa && " +
+			"/usr/sbin/sshd -f /opt/domino/etc/ssh/sshd_config -o \"Port 2223\" -De",
+	}
+	sidecarImage = "quay.io/ddl_apetrov/rsync-sidecar:latest"
```

Comment:
^^^ To be moved to a more appropriate location

Reply:
Some notes for productionizing this (cc @JoshBroomberg):

* We should add a GitHub action to publish the images using the same scheme as DCO itself. 
* We will need to figure out how to configure the image the DCO uses. I believe we will need to propagate this config from Helm -> DCO CLI. 
* We need to add this image to the catalog of images in platform apps which is used for pulling images into internal registries during offline installs. 

@adp312 Can you generate some follow-up tickets for this?